ui/app/controllers/vault/cluster/access/methods.js

```diff
@@ -85,4 +87,7 @@ export default class VaultClusterAccessMethodsController extends Controller {
       this.methodToDisable = null;
     }
   }
+
+  // template helper
+  sortMethods = (methods) => sortObjects(methods.slice(), 'path');
```

Comment:
made this a local helper instead of an app-wide one because sorting typically happens in the lazyPaginatedQuery method (which this route doesn't use) and so there didn't seem to be a need for this app-wide